Disable the test suite.

Last-Update: 2015-08-13

--- a/Makefile.am
+++ b/Makefile.am
@@ -31,7 +31,7 @@
 cpp_pc_file = poppler-cpp.pc
 endif
 
-SUBDIRS = goo fofi $(splash_subdir) poppler utils $(glib_subdir) test $(qt4_subdir) $(qt5_subdir) $(cpp_subdir)
+SUBDIRS = goo fofi $(splash_subdir) poppler utils $(glib_subdir) $(qt4_subdir) $(qt5_subdir) $(cpp_subdir)
 
 EXTRA_DIST =					\
 	README-XPDF				\
