Use dummy implementation of mutexes.

Last-Update: 2022-08-02

--- a/poppler/Annot.h
+++ b/poppler/Annot.h
@@ -53,7 +53,7 @@
 
 #include <memory>
 #include <atomic>
-#include <mutex>
+#include <dummy-mutex>
 #include <vector>
 
 #include "AnnotStampImageHelper.h"
--- a/poppler/Array.h
+++ b/poppler/Array.h
@@ -29,7 +29,7 @@
 #define ARRAY_H
 
 #include <atomic>
-#include <mutex>
+#include <dummy-mutex>
 #include <vector>
 
 #include "poppler-config.h"
--- a/poppler/CairoFontEngine.h
+++ b/poppler/CairoFontEngine.h
@@ -32,7 +32,7 @@
 #define CAIROFONTENGINE_H
 
 #include <memory>
-#include <mutex>
+#include <dummy-mutex>
 #include <unordered_map>
 #include <vector>
 
--- a/poppler/Dict.h
+++ b/poppler/Dict.h
@@ -31,7 +31,7 @@
 #define DICT_H
 
 #include <atomic>
-#include <mutex>
+#include <dummy-mutex>
 #include <string>
 #include <vector>
 #include <utility>
--- a/poppler/GlobalParams.h
+++ b/poppler/GlobalParams.h
@@ -47,7 +47,7 @@
 #include <unordered_map>
 #include <string>
 #include <memory>
-#include <mutex>
+#include <dummy-mutex>
 #include <vector>
 
 class GooString;
--- a/poppler/PDFDoc.h
+++ b/poppler/PDFDoc.h
@@ -49,7 +49,7 @@
 
 #include <algorithm>
 #include <cstdio>
-#include <mutex>
+#include <dummy-mutex>
 
 #include "poppler-config.h"
 
--- a/poppler/Page.h
+++ b/poppler/Page.h
@@ -36,7 +36,7 @@
 #define PAGE_H
 
 #include <memory>
-#include <mutex>
+#include <dummy-mutex>
 
 #include "poppler-config.h"
 #include "Object.h"
