Adapt get_poppler_datadir() to pdf2djvu needs.

Last-Update: 2021-10-04

--- a/poppler/GlobalParams.cc
+++ b/poppler/GlobalParams.cc
@@ -131,15 +131,10 @@ static const char *get_poppler_datadir(v
         return retval;
 
     if (!GetModuleFileNameA(hmodule, (CHAR *)retval, sizeof(retval) - 20))
-        return POPPLER_DATADIR;
+        abort();
 
     p = _mbsrchr((unsigned char *)retval, '\\');
     *p = '\0';
-    p = _mbsrchr((unsigned char *)retval, '\\');
-    if (p) {
-        if (stricmp((const char *)(p + 1), "bin") == 0)
-            *p = '\0';
-    }
     strcat(retval, "\\share\\poppler");
 
     beenhere = 1;
