Adapt get_poppler_datadir() to pdf2djvu needs.

Last-Update: 2018-01-17

--- a/poppler/GlobalParams.cc
+++ b/poppler/GlobalParams.cc
@@ -173,11 +173,6 @@
 
   p = _mbsrchr ((unsigned char *) retval, '\\');
   *p = '\0';
-  p = _mbsrchr ((unsigned char *) retval, '\\');
-  if (p) {
-    if (stricmp ((const char *) (p+1), "bin") == 0)
-      *p = '\0';
-  }
   strcat (retval, "\\share\\poppler");
 
   beenhere = 1;
