Adapt get_poppler_datadir() to pdf2djvu needs.

Last-Update: 2022-08-02

--- a/poppler/GlobalParams.cc
+++ b/poppler/GlobalParams.cc
@@ -126,11 +126,11 @@ BOOL WINAPI DllMain(HINSTANCE hinstDLL,
 
 static std::string get_poppler_localdir(const std::string &suffix)
 {
-    const std::string binSuffix("\\bin");
+    const std::string binSuffix("");
     std::string retval(MAX_PATH, '\0');
 
     if (!GetModuleFileNameA(hmodule, retval.data(), retval.size())) {
-        return POPPLER_DATADIR;
+        abort();
     }
 
     const std::string::size_type p = retval.rfind('\\');
