Adapt get_poppler_datadir() to pdf2djvu needs.

Last-Update: 2021-05-04

--- a/poppler/GlobalParams.cc
+++ b/poppler/GlobalParams.cc
@@ -135,11 +135,6 @@ static const char *get_poppler_datadir(v
 
     p = _mbsrchr((unsigned char *)retval, '\\');
     *p = '\0';
-    p = _mbsrchr((unsigned char *)retval, '\\');
-    if (p) {
-        if (stricmp((const char *)(p + 1), "bin") == 0)
-            *p = '\0';
-    }
     strcat(retval, "\\share\\poppler");
 
     beenhere = 1;
