Adapt get_poppler_datadir() to pdf2djvu needs.

Last-Update: 2015-08-13

--- a/poppler/GlobalParams.cc
+++ b/poppler/GlobalParams.cc
@@ -169,16 +169,11 @@
     return retval;
 
   if (!GetModuleFileNameA (hmodule, (CHAR *) retval, sizeof(retval) - 20))
-    return POPPLER_DATADIR;
+    strcpy(retval, ".\\.");
 
   p = _mbsrchr ((unsigned char *) retval, '\\');
   *p = '\0';
-  p = _mbsrchr ((unsigned char *) retval, '\\');
-  if (p) {
-    if (stricmp ((const char *) (p+1), "bin") == 0)
-      *p = '\0';
-  }
-  strcat (retval, "\\share\\poppler");
+  strcat (retval, "\\poppler-data");
 
   beenhere = 1;
 
