Don't use the GM_BUILD_* macros.

Last-Update: 2015-08-21

--- a/magick/command.c
+++ b/magick/command.c
@@ -17126,6 +17126,7 @@ static MagickPassFail VersionCommand(Ima
 #endif /* defined(HasZLIB) */
   PrintFeature("ZLIB", supported);
 
+#if 0
 #if defined(GM_BUILD_HOST)
   (void) fprintf(stdout,"\nHost type: %.1024s\n", GM_BUILD_HOST);
 #endif /* defined(GM_BUILD_HOST) */
@@ -17163,6 +17164,7 @@ static MagickPassFail VersionCommand(Ima
 #if defined(GM_BUILD_LIBS)
   (void) fprintf(stdout,"  LIBS     = %.1024s\n", GM_BUILD_LIBS);
 #endif /* defined(GM_BUILD_LIBS) */
+#endif
 
 #if defined(_VISUALC_)
 
