Get rid of EXIV2BIN; use EXIV2EXE everywhere.

Forwarded: http://dev.exiv2.org/issues/1102

--- a/src/Makefile
+++ b/src/Makefile
@@ -174,7 +174,6 @@
 
 EXIV2OBJ  = $(EXIV2MAIN:.cpp=.o) $(EXIV2SRC:.cpp=.o)
 EXIV2COBJ = $(EXIVCSRC:.c=.o)
-EXIV2BIN  = $(EXIV2MAIN:.cpp=)
 EXIV2EXE  = $(EXIV2MAIN:.cpp=$(EXEEXT))
 
 ifdef DEP_TRACKING
@@ -240,7 +239,7 @@
 $(LIBTOOL): $(LIBTOOL_DEPS)
 	$(SHELL) $(top_srcdir)/config.status --recheck
 
-bin: lib $(EXIV2BIN)
+bin: lib $(EXIV2EXE)
 
 lib: $(OBJ)
 	$(LIBTOOL) --mode=link $(LINK.cc) -o $(LIBRARY) $(LOBJ) -rpath $(libdir) -version-info $(EXIV2_LTVERSION) $(LIBS)
@@ -249,7 +248,7 @@
 $(BINARY): %: %.o lib
 	@$(LIBTOOL) --mode=link $(LINK.cc) -o $@ $(LIBRARY) $@.o -rpath $(libdir)
 
-$(EXIV2BIN): lib $(EXIV2OBJ) $(EXIV2COBJ)
+$(EXIV2EXE): lib $(EXIV2OBJ) $(EXIV2COBJ)
 	mkdir -pv ../bin 2>&1 > /dev/null
 	@$(LIBTOOL) --mode=link $(LINK.cc) -o ../bin/$@ $(LIBRARY) $(EXIV2OBJ) $(EXIV2COBJ) -rpath $(libdir)
 
@@ -273,7 +272,7 @@
 	$(INSTALL_DIRS) $(DESTDIR)$(libdir)/pkgconfig
 	$(INSTALL_DATA) $(top_srcdir)/config/exiv2.pc $(DESTDIR)$(libdir)/pkgconfig/exiv2.pc
 
-install: $(EXIV2BIN) install-lib
+install: $(EXIV2EXE) install-lib
 	$(INSTALL_DIRS) $(DESTDIR)$(bindir)
 	@$(LIBTOOL) --mode=install $(INSTALL_PROGRAM) ../bin/$(EXIV2EXE) $(DESTDIR)$(bindir)/$(EXIV2EXE)
 	$(INSTALL_DIRS) $(DESTDIR)$(mandir)
