Pass -Wno-obsolete to automake.

Lat-Update: 2015-08-13

--- a/configure.ac
+++ b/configure.ac
@@ -21,7 +21,7 @@
 
 # Initialize Automake
 # Don't require all the GNU mandated files
-AM_INIT_AUTOMAKE([-Wall -Werror ansi2knr no-dist foreign])
+AM_INIT_AUTOMAKE([-Wall -Werror -Wno-obsolete ansi2knr no-dist foreign])
 
 # Make --enable-silent-rules the default.
 # To get verbose build output you may configure
